be/src/column/object_column.h

```diff
@@ -22,7 +22,7 @@ namespace starrocks::vectorized {
 //};
 
 template <typename T>
-class ObjectColumn final : public ColumnFactory<Column, ObjectColumn<T>> {
+class ObjectColumn : public ColumnFactory<Column, ObjectColumn<T>> {
```

Comment:
Why delete final?

Reply:
The `JsonColumn` inherits `ObjectColumn` to reuse most of its implementation, so final is removed.